static/styles/zulip.css

```diff
@@ -280,6 +280,9 @@ p.n-margin {
     position: relative;
 
     .alert {
+        /* As we're now only keeping active navbar alert in the DOM.
+        We don't need the default alert's property to hide it now. */
```

Comment:
Generally "now" isn't a word that should appear in a comment, as that will read oddly for someone looking at this code in 2 years.  We should just explain that we override the default hidden CSS for the `alert` class because we only include these elements in the DOM when we want them displayed.

Reply:
Okay! Will keep it in mind.